itou/metabase/management/commands/populate_metabase_itou.py

```diff
@@ -308,21 +309,28 @@ def populate_selected_jobs(self):
         Populate associations between job applications and job descriptions.
         """
         table_name = "fiches_de_poste_par_candidature"
-        self.log(f"Preparing content for {table_name} table...")
+        chunk_size = 1000
+        self.log(f"Preparing content for {table_name} table by chunk of {chunk_size} items...")
+
+        # Iterating directly on this very large queryset results in psycopg2.errors.DiskFull error.
```

Comment:
J'ai du mal à comprendre pourquoi l'[erreur Sentry mentionnée expose une erreur d'assertion](https://sentry.io/organizations/betagouv-f7/issues/2685512432/?project=5671910&query=is%3Aresolved&statsPeriod=14d) et qu'on parle ici d'un problème avec `psycopg2.errors.DiskFull` ?

D'autre part le `queryset` est stocké en mémoire, y a-t-il un rapport avec `psycopg2.errors.DiskFull` ?

Reply:
Un train peut en cacher un autre. Il y a en fait 2 erreurs. La première que j'ai résolu ici. La seconde remontée par Sentry que j'attaque maintenant.

> D'autre part le queryset est stocké en mémoire, y a-t-il un rapport avec psycopg2.errors.DiskFull ?
C'est surprenant hein ? Qu'une lecture d'un gros queryset cause un remplissage de disque. Je suis aussi étonné que toi. Mais le fait est, et ce de façon reproductible, que sans pagination du queryset on prend un DiskFull, et avec pagination plus de problème.

Reply:
Ok, c'est peut être la mémoire virtuelle mais ça paraît bizarre comme ça. C'est sur quelle machine ?

Reply:
Pour info, l'autre erreur (celle de sentry) était finalement juste une donnée pourrie que j'ai fixé manuellement dans l'admin. Pas besoin de code.
Cf thread slack https://itou-inclusion.slack.com/archives/C01181Y04LT/p1633353247260800